ui-core/src/participant/I18nProvider.tsx

```diff
@@ -32,12 +32,15 @@ const SELECTED_LANGUAGE_KEY = 'selectedLanguage'
 /**
  * Provider for the current users i18n context.
  */
-export function I18nProvider({ portalShortcode, children }: { portalShortcode?: string, children: React.ReactNode }) {
+export function I18nProvider({ defaultLanguage, portalShortcode, children }: {
+  defaultLanguage: string, portalShortcode?: string, children: React.ReactNode
+}) {
   const Api = useApiContext()
   const [isLoading, setIsLoading] = useState(true)
   const [isError, setIsError] = useState(false)
   const [languageTexts, setLanguageTexts] = useState<Record<string, string>>({})
-  const [selectedLanguage, setSelectedLanguage] = useState(localStorage.getItem(SELECTED_LANGUAGE_KEY) || 'en')
+  const [selectedLanguage, setSelectedLanguage] = useState(
+    localStorage.getItem(SELECTED_LANGUAGE_KEY) || defaultLanguage)
```

Comment:
That's a great point- the backend _can_ take nulls for portal-related endpoints, but not for the language texts. Since most of those are global, there's not really an equivalent concept of defaultLanguage, which is why I've just settled on English. Fortunately the number of places where this is hardcoded are minimal but I hear you that it would be better to have _none_